markdown/ch3.markdown

```diff
@@ -69,19 +69,23 @@ with matplotlib. First, we import the necessary packages, and use the `matplotli
 inline` IPython magic to make our images appear below the code:
 
 ```python
+# Make plots appear inline, set custom plotting style
 %matplotlib inline
+import matplotlib.pyplot as plt
+plt.style.use('style/elegant.mplstyle')
+```
+
+```python
 import numpy as np
 import matplotlib as mpl
-from matplotlib import pyplot as plt
 from matplotlib import cm  # colormap module
-plt.style.use('style/elegant.mplstyle')
 ```
 
 Finally, "make some noise" and display it as an image:
 
 ```python
 random_image = np.random.rand(500, 500)
-plt.imshow(random_image);
+plt.imshow(random_image); # ; suppresses the output (i.e. the plotting object text) being displayed in the notebook.
```

Comment:
This is now probably too wide... Maybe make it a 2 line comment underneath the call? I suggest:

```python
# the semicolon (;) prevents the return value of the `imshow` call, which is a
# matplotlib Axes object, from being printed as a string, in addition to displayed.